### DISCUSSION TITLE
Downloaded repository doesn't load

### DISCUSSION THREAD

**Discussion by polarizeproject** [community/community#40002] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have both tried to clone and to download this music visualizer: [ playways](https://github.com/katshaze/playways)
I can see it works online but I can't seem to make it work on local, it opens but it's stuck on Loading and doesn't play the songs. 

What can I do to have it fully working? The files are the same, it's the same code so I don't get why the downloaded version isn't working... 
Can anyone help? Thanks!

EDIT: All these errors keep showing up in the local console:

![Senza-nome](https://user-images.githubusercontent.com/107323117/203837684-b49e4968-7f54-489b-a7b5-7069d157487c.png)

**Comment by Preetiraj3697** [comment]
Go to this page https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors
This show why this issue is coming and how to solve this issue. 😊

**Reply by polarizeproject** [reply]
I followed the steps and cloned the repository through ssh key but I still have the same issue :(
